shell/browser/electron_browser_main_parts.cc

```diff
@@ -242,7 +231,6 @@ void ElectronBrowserMainParts::RegisterDestructionCallback(
 int ElectronBrowserMainParts::PreEarlyInitialization() {
   field_trial_list_ = std::make_unique<base::FieldTrialList>(nullptr);
 #if defined(OS_LINUX)
```

Comment:
The `if defined` can also be removed.

Reply:
I'm afraid, there is some misunderstandings
1. while new userCache path is not modified, the location remains the same, it is **backward compatible**.
2. if as an Electron app developer I want to control where the whole cache is located (cookies, db, session, file cache..), this is under my responsibility. I do not get the point to split the cache into different pieces. From my point of view, it is less error-prone, simpler and more consistent, to consider all 'temporary' files as a whole. my2cents

Reply:
done